src/components/OcAccordion.spec.js

```diff
@@ -0,0 +1,119 @@
+import OcAccordion from './OcAccordion.vue';
+import {mount} from '@vue/test-utils';
+import OcAccordionItem from './OcAccordionItem.vue';
+import OcIcon from './OcIcon.vue';
+import OcButton from './OcButton.vue';
+
+const stubs = {
+  'oc-accordion-item': OcAccordionItem,
+  'oc-button': OcButton,
+  'oc-icon': OcIcon
+}
+
+const singleAccordionItemSlot = '<oc-accordion-item title="My accordion item" icon="folder">' +
+  '<p>I am the content of this accordion</p>' +
+  '</oc-accordion-item>'
+
+const multipleItemsSlot = '<oc-accordion-item title="First accordion item">' +
+  '<p class="first-item-message">Hello there first.</p>' +
+  '</oc-accordion-item>' +
+  '<oc-accordion-item title="Second accordion item">' +
+  '<p class="second-item-message">Hello here second.</p>' +
+  '</oc-accordion-item>'
+
+describe('OcAccordion', () => {
+  /**
+   * @param {Object} props
+   * @param {string} slot
+   * @returns {Wrapper<Vue>}
+   */
+  function getWrapperWithSlotAndProps(props, slot=singleAccordionItemSlot) {
+    return mount(OcAccordion, {
+      slots: {
+        default: slot
+      },
+      propsData: {
+        ...props
+      },
+      stubs
+    })
+  }
+  describe("when item title is clicked", () => {
```

Comment:
let's use single quote `' ` everywhere. I'm surprised the linter does not catch that

Reply:
spec files are ignored for linting. i've added a PR to unskip them
https://github.com/owncloud/owncloud-design-system/pull/1391

Reply:
double quotes is used everywhere now!